cli/action.go

```diff
@@ -102,5 +105,36 @@ func action(c *cli.Context) error {
 
 	log.Infof("bitrise secrets generated at: %s", secretsPth)
 
+	if err := gitignore(".bitrise.secrets.yml", "./.gitignore"); err != nil {
+		log.Warnf("Could not add .bitrise.secrets.yml to .gitignore: %s", err)
+		log.Warnf("Please be advised, that for security considerations, it is not recommended to upload .bitrise.secrets.yml to version control")
+	}
+
+	return nil
+}
+
+func gitignore(pattern, gitignorePath string) error {
+	f, err := os.OpenFile(gitignorePath, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
+	if err != nil {
+		return fmt.Errorf("open .gitignore file at %s: %s", gitignorePath, err)
+	}
+
+	contents, err := ioutil.ReadFile(gitignorePath)
```

Comment:
122-129 is a block that should avoid opening gitignorePath in line 117

recommended to re-order the logic here

Reply:
AFAIK we discussed that we won't have existence check for now, just a single fopen with ACW.

Reply:
there is a way to not to open a file 2 times in a single block using f for scanning and checking lines: https://stackoverflow.com/questions/37255304/golang-find-string-in-file-and-show-line-number/37269433

and after the check you can reset the f io seeker

Reply:
that would avoid regex usage